Update api.md (#1015)
jmorganca/ollama: docs/api.md

- `eval_count`: number of tokens the response
- `eval_duration`: time in nanoseconds spent generating the response
- `context`: an encoding of the conversation used in this response, this can be sent in the next request to keep a conversational memory

> - `response`: one token if the response was streamed, if not streamed, this will contain the full response

Reviewer: ```suggestion
- `response`: the last token if the response was streamed, if `stream` is `false`, this will contain the full response
```
